### PR TITLE
fix(doc-core): the 1440px device displays a horizontal scroll bar

### PR DESCRIPTION
**Pull request by zhousg**
## Summary

image

### <samp>🤖 Generated by Copilot at 02bd1d3</samp>

Adjust width calculations of `docLayout` and `content` elements in default theme for documentation generator. Prevent horizontal overflow and improve responsiveness on large screens.

### <samp>🤖 Generated by Copilot at 02bd1d3</samp>

* Fix docLayout and content width overflow on large screens ([link](https://github.com/web-infra-dev/modern.js/pull/3591/files?diff=unified&w=0#diff-bba1d4ce4b34efc0704505701ad0803d218232fb20f86383cd9e268f7aed7718L57-R57), [link](https://github.com/web-infra-dev/modern.js/pull/3591/files?diff=unified&w=0#diff-bba1d4ce4b34efc0704505701ad0803d218232fb20f86383cd9e268f7aed7718L63-R63))

## Checklist

- [x] I have added changeset via `pnpm run change`.
- [ ] I have updated the documentation.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: .changeset/fifty-years-judge.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@modern-js/doc-core': major
```
Comment (sanyuan0704) [inline .changeset/fifty-years-judge.md:2]: Remove it

File: packages/cli/doc-core/src/theme-default/layout/DocLayout/index.module.scss [hunk 2]
```diff
@@ -54,13 +54,13 @@
 
 @media (min-width: 1440px) {
   .docLayout {
-    width: 1440px;
+    width: calc(1440px - 100vw + 100%);
```
Comment (sanyuan0704) [inline packages/cli/doc-core/src/theme-default/layout/DocLayout/index.module.scss:57]: Why `-100vw + 100%`?

Comment (zhousg) [reply packages/cli/doc-core/src/theme-default/layout/DocLayout/index.module.scss:57]: 100vw is viewport ，100% is body width  ，the rest is the width of the scroll bar

**Comment by sanyuan0704** [comment]
I found it doesn't appear horizental scroll bar in 1440px screen:

image

How to reproduce the bug?

**Comment by zhousg** [comment]
> I found it doesn't appear horizental scroll bar in 1440px screen:
> 
> image
> 
> How to reproduce the bug?

If you connect to a secondary screen

**Comment by sanyuan0704** [comment]
I just conneced a extra screen, but it hasn't reproduce :)

**Comment by zhousg** [comment]
image
this can reproduce

**Comment by sanyuan0704** [comment]
Get, it's good advice!By the way the 960px ~ 1440px will also reproduce it.

**Review by sanyuan0704 — APPROVED** [review]

**Comment by sanyuan0704** [comment]
The changset file should add chinese version

**Review by sanyuan0704 — APPROVED** [review]

**Pull request merged**